util/verible-format-allowlist.vbl

```diff
@@ -0,0 +1,18 @@
+# Copyright lowRISC contributors.
```

Comment:
Let's make this a file with a `.txt` suffix or so, `.vbl` is for Verible waiver/config files which are consumed by Verible directly

Reply:
Changed to `.txt`.